domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -119,51 +128,94 @@ sealed class Søknadsbehandling : BehandlingMedOppgave, BehandlingMedAttestering
         return Unit.right()
     }
 
-    fun leggTilFradragsgrunnlag(
+    fun leggTilFradragsgrunnlagFraSaksbehandler(
         saksbehandler: NavIdentBruker.Saksbehandler,
         fradragsgrunnlag: List<Grunnlag.Fradragsgrunnlag>,
+        clock: Clock,
     ) = vilkårsvurder(saksbehandler).let {
         when (it) {
-            is KanBeregnes -> leggTilFradragsgrunnlagInternal(saksbehandler, fradragsgrunnlag)
+            is KanBeregnes -> leggTilFradragsgrunnlagInternalForSaksbehandler(saksbehandler, fradragsgrunnlag, clock)
             else -> KunneIkkeLeggeTilGrunnlag.KunneIkkeLeggeTilFradragsgrunnlag.IkkeLovÅLeggeTilFradragIDenneStatusen(
                 this::class,
             ).left()
         }
     }
 
-    private fun leggTilFradragsgrunnlagInternal(
+    protected fun leggTilFradragsgrunnlagForBeregning(
         saksbehandler: NavIdentBruker.Saksbehandler,
```

Comment:
Se kommentar nedenfor angående saksbehandler for operasjoner utført av systemet.